boards/sensebox_samd21/include/board.h

```diff
@@ -123,24 +123,33 @@ extern "C" {
  * merged DIO0 and DIO1 interupt pins into one (defined as DIOMULTI).
  * @{
  */
-#define SX127X_PARAM_SPI                    (SPI_DEV(0))
+#define SX127X_PARAM_SPI            (SPI_DEV(0))
 
-#define SX127X_PARAM_SPI_NSS                XBEE1_CS_PIN        /* D23 */
+#define SX127X_PARAM_SPI_NSS        XBEE1_CS_PIN        /* D23 */
 
-#define SX127X_PARAM_RESET                  GPIO_UNDEF
+#define SX127X_PARAM_RESET          GPIO_UNDEF
 
-#define SX127X_PARAM_DIOMULTI               XBEE1_INT_PIN       /* D24 */
+#define SX127X_PARAM_DIO0           GPIO_UNDEF
 
-#define SX127X_PARAM_PASELECT               (SX127X_PA_RFO)
+#define SX127X_PARAM_DIO1           GPIO_UNDEF
 
-#define SX127X_PARAMS                     { .spi       = SX127X_PARAM_SPI,     \
-                                            .nss_pin   = SX127X_PARAM_SPI_NSS, \
-                                            .reset_pin = SX127X_PARAM_RESET,   \
-                                            .dio0_pin  = SX127X_PARAM_DIOMULTI,\
-                                            .dio1_pin  = SX127X_PARAM_DIO1,    \
-                                            .dio2_pin  = SX127X_PARAM_DIO2,    \
-                                            .dio3_pin  = SX127X_PARAM_DIO3,    \
-                                            .paselect  = SX127X_PARAM_PASELECT }
+#define SX127X_PARAM_DIO2           GPIO_UNDEF
+
+#define SX127X_PARAM_DIO3           GPIO_UNDEF
+
+#define SX127X_PARAM_DIOMULTI       XBEE1_INT_PIN       /* D24 */
+
+#define SX127X_PARAM_PASELECT       (SX127X_PA_RFO)
+
+#define SX127X_PARAMS               { .spi       = SX127X_PARAM_SPI,           \
```

Comment:
This define is not needed since all `SX127X_PARAMS_*` are already defined and will be used in the `sx127x_params.h` file.